submissions/cynthiaNwakaeme/domManipulation/shippingCost.js

```diff
@@ -0,0 +1,48 @@
+let quantity = document.getElementById("quantityNumber"),
+    price = document.getElementById("price"),
+    shippingCost = document.getElementById("shippingCost"),
+    totalCost = document.getElementById("totalCost"),
+    button = document.getElementById("buyNow");
+
+
+
+let shippingAnalysis = (event) => {
+    if (quantity.value == "" || quantity.value <= 0) {
```

Comment:
This `if` block can listen for `submit` event when the quantity is empty

Reply:
I disabled the button . There might not be a need for that or what do you think?

Reply:
ok